contrib/pyinstaller_spec.spec

```diff
@@ -41,23 +41,17 @@ a = Analysis(
 
 pyz = PYZ(a.pure, a.zipped_data, cipher=None)
 
-exe = EXE(
-    pyz,
-    a.scripts,
-    exclude_binaries=True,
-    name="cli",
-    debug=False,
-    strip=False,
-    upx=True,
-    console=True,
-)
-
-coll = COLLECT(
-    exe,
-    a.binaries,
-    a.zipfiles,
-    a.datas,
-    strip=False,
-    upx=True,
-    name="fava",
-)
+exe = EXE(pyz,
+          a.scripts,
+          a.binaries,
+          a.zipfiles,
+          a.datas,
+          [],
+          name='fava',
+          debug=False,
```

Comment:
Can you remove all the kwargs that, just like this one, simply correspond to the default value?

Reply:
done... I removed unnecessary kwargs in 5aa256e44016e774febca7cd8dd1b94105d69e2e